sass/style.scss

```diff
@@ -6,3 +6,4 @@
 @import 'header';
 @import 'content';
 @import 'footer';
+@import 'fonts';
```

Comment:
You will want this import statement towards the top. The font need to be loaded before you can use them! 
So put the @import fonts before the header import.